src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -131,7 +134,7 @@ export class MdAutocompleteTrigger implements AfterContentInit, ControlValueAcce
    * A stream of actions that should close the autocomplete panel, including
    * when an option is selected and when the backdrop is clicked.
    */
-  get panelClosingActions(): Observable<any> {
+  get panelClosingActions(): Observable<MdOptionSelectEvent | null> {
```

Comment:
Just thought we should start so that it's more explicit. It's not super important to change in this PR, so I can revert for now.